recipe/build.sh

```diff
@@ -37,6 +37,10 @@ elif [[ "$target_platform" == osx-* ]]; then
   ln -sf $PREFIX/bin/ld $PREFIX/libexec/pocl/ld
 fi
 
+if [[ "$target_platform" == linux-ppc64le ]]; then
+  EXTRA_HOST_CLANG_FLAGS="${EXTRA_HOST_CLANG_FLAGS} -faltivec-src-compat=mixed"
```

Comment:
According to https://clang.llvm.org/docs/ClangCommandLineReference.html, it looks like `mixed` is the default.

Reply:
Yes, but the warning is about a change in the default to `xl`.